shared/actions/wallets.js

```diff
@@ -41,12 +44,64 @@ const loadPaymentsSuccess = (res: RPCTypes.PaymentsPageLocal, action: WalletsGen
   )
 }
 
+const linkExistingAccount = (state: TypedState, action: WalletsGen.LinkExistingAccountPayload) => {
+  const {name, secretKey} = action.payload
+  return RPCTypes.localLinkNewWalletAccountLocalRpcPromise(
+    {
+      name,
+      secretKey: secretKey.stringValue(),
+    },
+    Constants.linkExistingWaitingKey
+  )
+    .then(accountID => [
```

Comment:
this might not work after my change to util/saga thats in pr. This thing is supposed to return a single thing to yield, so returning an array should give you a warning, instead we should be using sequentially